web/src/cypress/support/page-fab.ts

```diff
@@ -12,19 +12,30 @@ declare global {
 }
 
 function pageFab(dialOption?: string): Cypress.Chainable {
-  const res = cy.get('button[data-cy=page-fab]').should('be.visible')
-
   // standard page fab
-  if (!dialOption) return res.click()
+  if (!dialOption)
+    return cy
+      .get('button[data-cy=page-fab]')
+      .should('be.visible')
+      .then(res => {
+        res.click()
+      })
```

Comment:
No need for `.then` it will already be chained/queued.
```suggestion
      .click()
```